### ISSUE TITLE
Cannot reorder 2sxc modules

### ISSUE THREAD

**Issue by EzraM** [2sic/2sxc#855]
I'm running into an issue reordering 2sxc modules on a page.
This is on  DNN 8.0.2

I've made 3 pages.

Page 1: HTML modules. Can reorder.
Page 2: 2sxc modules. Cannot reorder.
Page 3: 2 HTML modules, 1 2sxc module. Can reorder HTML modules. Cannot reorder 2sxc modules.

Some notes:
The MoveModule callback looks ok for all three pages and all modules.
When the page reloads, however, the 2sxc module's location on the page is not updated.

Skin is a full-width div.
Container is also a blank div.

Let me know if there's other things I could check to narrow down this issue.

Thanks!

**Comment by iJungleboy** [comment]
I believe this is a DNN bug, as 2sxc modules are not "special" in any way. I've experienced similar stuff but ignored it as I was able to work around it quickly, but to 99% it's a DNN issue; 2sxc is not involved when moving a module. 

I also remember not being able to move the module no another pane on the same page - so I'm certain it's a dnn issue - pls check the DNN jira and what DNN does in the DB when it receives the re-order call. 

Often this could also be a DNN-caching issue. 

If you find out I'm wrong - just comment it here and I'll re-open.
